ajax/libs/tern/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "tern",
+  "license": "MIT",
+  "version": "0.18.0",
+  "author": "Marijn Haverbeke <marijnh@gmail.com>",
+  "description": "A JavaScript code analyzer for deep, cross-editor language support",
+  "filename": "lib/tern.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ternjs/tern.git"
+  },
+  "dependencies": {
+    "acorn": "^2.7.0",
+    "enhanced-resolve": "^0.9.1",
+    "glob": "3",
+    "minimatch": "0.2",
+    "typescript": "=1.0.1"
+  },
+  "homepage": "http://ternjs.net/",
+  "keywords": [
+    "stand-alone",
+    "editor-independent",
+    "JavaScript",
+    "analyzer",
+    "integration"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ternjs/tern.git",
+    "basePath": "",
+    "files": [
+      "lib/(comment*|condense*|def*|infer*|signal*|tern*)+.(js|map)",
+      "plugin/(angular*|commonjs*|complete_strings*|doc_comment*|es_modules*|modules*|node*|node_resolve*|requirejs*|webpack*)+.(js|map)"
```

Comment:
hmmm... i'm not sure, do i need to ask the author, or i just remove those files.